processor/elasticapmprocessor/testdata/elastic_hostname/spanss_output.yaml

```diff
@@ -0,0 +1,63 @@
+resourceSpans:
```

Comment:
is the file rename intended?
it's moving from `spans_output.yaml` to `spanss_output.yaml`